api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters-blm/marxan-sandbox-blm-runner.service.ts

```diff
@@ -80,6 +88,18 @@ export class MarxanSandboxBlmRunnerService
               marxanRun.stdError,
             );
 
+            await this.blmPartialRepository.save(
+              output,
+              forScenarioId,
+              blmValue,
+            );
+            await this.blmFinalRepository.saveBest(
+              output,
+              calibrationId,
+              forScenarioId,
+              blmValue,
+            );
```

Comment:
[blocker] Please read the .md docs. We shouldn't save anything in `final` repository until all partial results are gathered.